### ISSUE TITLE
Add LMS icons

### ISSUE THREAD

**Issue by brian-smith-tcril** [openedx/paragon#3024]
* Get a link with all the icons so we know what needs to be added
  * https://github.com/raccoongang/frontend-app-learning/tree/sidebar-main/src/courseware/course/sidebar/sidebars/course-outline/icons
* Add new “LMS Icons” page similar to https://paragon-openedx.netlify.app/foundations/brand-icons/ - add these icons there
* Note for usage: screen reader text should be complete-name as opposed to name-complete
